corehq/apps/reports/standard/deployments.py

```diff
@@ -98,9 +70,16 @@ def headers(self):
                              sort_type=DTSortType.NUMERIC),
             DataTablesColumn(_("Last Sync"),
                              sort_type=DTSortType.NUMERIC),
-            DataTablesColumn(_("Application (Deployed Version)"),
-                help_text=_("""Displays application version of the last submitted form;
-                            The currently deployed version may be different."""))
+            DataTablesColumn(_("Application"),
+                             help_text=_("Displays application of last submitted form")),
+            DataTablesColumn(_("Application Version"),
+                             help_text=_("""Displays application version of the last submitted form;
+                                         The currently deployed version may be different."""),
```

Comment:
Maybe? but it doesn't matter once it gets down to HTML since whitespace isn't significant there. I just copied this way of doing it from the old Application Deployed Version column

![image](https://cloud.githubusercontent.com/assets/1471773/21016030/235d2dac-bd32-11e6-9b23-705607d505c4.png)